### ISSUE TITLE
Dengo plug&play controller convert issue

### ISSUE THREAD

**Issue by 2041816** [GMMan/dengo-plug-and-play-controller#1]
Hello, i wanna to ask that what is the "door light" meaning to? I even cant find out this in my controller, and i want ask why it still didnt work when i cloned repo of this content of tutorial post to USB Driver, Thanks. Good day

**Comment by GMMan** [comment]
The door light is the big domed indicator light on the unit's panel, beside the arrow buttons.

**Comment by 2041816** [comment]
and i want ask why it still didnt work when i cloned repo of this content of tutorial post to USB Driver, Thanks too
.

**Comment by 2041816** [comment]
https://imgur.com/ek2UCo1
Is my way of placing the files wrong?

**Comment by GMMan** [comment]
The files need to be extracted to the root of the drive, not nested inside a folder.

**Comment by 2041816** [comment]
is like this?
https://imgur.com/uFR1tUW

**Comment by GMMan** [comment]
Yes, if that's in the root of your drive it should be good.

**Comment by 2041816** [comment]
payload needed to be extracted?

**Comment by GMMan** [comment]
Looks like you already did?

**Comment by 2041816** [comment]
nope, but the door light still havn't turn on, dont know how slove it :0(

**Comment by 2041816** [comment]
my USB OTG Hub and USB Is working

**Comment by GMMan** [comment]
Drive is formatted FAT32? You may also want to clone the repo with Git instead of downloading a .zip of the code, as there are potential line ending issues if using the .zip.

**Comment by 2041816** [comment]
Yes, it's FAT32, Also, how can i clone the repo with Git?

**Comment by GMMan** [comment]
Install Git, then read [this](https://git-scm.com/book/en/v2/Git-Basics-Getting-a-Git-Repository#_git_cloning)

**Comment by 2041816** [comment]
still not working on my controller :0(, should the Problem is out in my USB?

**Comment by 2041816** [comment]
https://imgur.com/eKYU4Ju
https://imgur.com/niKhlou

**Comment by GMMan** [comment]
Is the drive MBR or GPT?

**Comment by 2041816** [comment]
MBR

**Comment by GMMan** [comment]
If the script is being run, you'll see a log file on the USB drive.

Have you checked that the script line ending is in Unix line ending rather than Windows line ending?

**Comment by 2041816** [comment]
emm why the script didnt run when i done all the step which given by this post

**Comment by 2041816** [comment]
i can't see the log file

**Comment by GMMan** [comment]
If you're sure the line endings are correct, it's probably the adapter you're using to connect to the P&P. Some of the cheap ones have data pins that are too short, for example.

**Comment by lonchaney1000** [comment]
Hello, thanks a lot for your work, GMMan, a dream finally come true! :) 
I just have two questions:

Should the door light work when we play the PC game? Because it doesn't for me. 
Do you how we can use the controller to play to DDG 1 and 2?
I don't have any settings app to enable it for these games.

Thanks again!

**Comment by GMMan** [comment]
The door light will not work because the PC version does not provide that info to USB Type-2 controllers. For earlier DDG games, you can try [this](http://autotraintas.hariko.com/) to have it convert the input.

**Comment by lonchaney1000** [comment]
Thanks for your fast reply. I will try that soon and let you know. Thanks again!

**Comment by lonchaney1000** [comment]
I just tried with Densha de Go 2 and it works! :) Thanks a lot, gMMan!
